frontend/public/components/machine-config-pool.tsx

```diff
@@ -245,17 +246,28 @@ const MachineConfigPoolTableHeader = () => {
       props: { className: tableColumnClasses[3] },
     },
     {
-      title: 'Degraded',
+      title: 'Paused',
       props: { className: tableColumnClasses[4] },
     },
     {
-      title: '',
+      title: 'Degraded',
       props: { className: tableColumnClasses[5] },
     },
+    {
+      title: '',
+      props: { className: tableColumnClasses[6] },
+    },
   ];
 };
 MachineConfigPoolTableHeader.displayName = 'MachineConfigPoolTableHeader';
 
+const getMCPPausedState = (obj: MachineConfigPoolKind) => {
+  if (_.isUndefined(obj.spec?.paused)) {
+    return 'Unknown';
+  }
+  return obj.spec?.paused ? 'True' : 'False';
```

Comment:
If it's undefined, it's not paused. So this can be just
```suggestion
  return obj.spec?.paused ? 'True' : 'False';
```

I'd inline it below.

Reply:
@spadgett I started with that but noticed that the other status differentiates between unknown and false. When you create a new mcp you can see them all show the same result.

Reply:
@jcaianirh Conditions have different semantics. We want false here for paused because `spec.paused` is always treated as false if missing.

Reply:
@spadgett thanks for clearing that up!